### DISCUSSION TITLE
netbox topology views installation

### DISCUSSION THREAD

**Discussion by lasag-2511** [mattieserver/netbox-topology-views#307] [General]
Hi all, I have the below error when running command. 

 python3 manage.py migrate netbox_topology_views

  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
CommandError: No installed app with label 'netbox_topology_views'

**Comment by dreng** [comment]
First of all, you shouldn't have to migrate yourself. Or are you trying to develop? Second, it should be "python3 manage.py migrate", without the plugin name.

**Comment by mattieserver** [comment]
Make sure the plugin is installed (pip install netbox-topology-views)
Also like @dreng said, you should just run `python3 manage.py migrate`.

Also make sure that you venv is active if you use that (`source /opt/netbox/venv/bin/activate` , if you use the default from netbox itself)
